components/Budget/ExpenseSummary.jsx

```diff
@@ -0,0 +1,88 @@
+import { TouchableOpacity, Text, View, Image } from "react-native";
+import { useState } from "react";
+import Subtitle from "../TextStyles/Subtitle";
+import Icon from "react-native-vector-icons/FontAwesome";
+import ProfilePicture from "../ProfileElements/ProfilePicture";
+import ImageViewer from "./ImageViewer";
+
+export default function ExpenseSummary({ expense, isUser }) {
+    const [isDropdownOpen, setIsDropdownOpen] = useState(false);
+
+    const timeString = new Date(expense.time).toLocaleString();
+    return (
+        <View className="py-1">
+            <TouchableOpacity
+                className={`justify-between w-full ${
+                    expense.placeholder ? "bg-slate-50" : "bg-slate-200"
+                } rounded-lg p-2 border-2`}
+                onPress={() => setIsDropdownOpen(!isDropdownOpen)}
+            >
+                <View className="flex-row justify-between">
+                    <View className="flex-row items-center flex-1">
+                        <ProfilePicture
```

Comment:
can this get a grey outline for placeholder

Reply:
think being able to change outline colour is a good shout generally, could do some things with diff outlines for admins n that